src/components/ColumnTitle/ColumnFilter.tsx

```diff
@@ -108,19 +165,19 @@ export const ColumnFilter: FunctionComponent<ColumnFilterProps> = (props) => {
         onVisibleChange={handleVisibleChange}
     >
         <div
-            className={cn(styles.icon, { [styles.active]: !!filter })}
-            dangerouslySetInnerHTML={{ __html: filterIcon }}
+            className={cn(styles.icon, {[styles.active]: !!filter})}
```

Comment:
I believe we follow the rule about spacing on object literals https://eslint.org/docs/rules/object-curly-spacing, so this changes unnecessary. I reconfigure our eslint config to enforce this rule later.

Reply:
resolve